sensor_box/library/Drone_sensor_library.ino

```diff
@@ -254,7 +254,7 @@ void loop() {
     float blankNoise = 0;
 
     for (int i = 0; i < 100; i++) {
-        sensor.start();
+        sensor.start(); //Lines 257-268 are the same as lines 274-285. Why? WSchuler

```

Comment:
the first section is the ON measurement, the second section is the OFF measurement that is subtracted from the ON